src/core/software/pupil-player.md

```diff
@@ -314,6 +314,27 @@ You can compare `2d` and `3d` mapping results by creating two calibrations and g
 ### Developing your own Plugin
 To develop your own plugin see the [developer guide](/developer/core/overview "Pupil Core developer documentation").
 
+### Product-specific plugins
+Some plugins in Pupil Player _only_ load if they are able to be used with the product that created the recordings.
+
+Here is a list of what plugins are available based on the product that generated the recording:
+
+| Pupil Player Plugin | Pupil Core | Pupil Invisible | Pupil Mobile |
+| :-: | :--: | :-: | :-: |
+| Blink detection | :heavy_check_mark:  |  :heavy_minus_sign: | :heavy_check_mark: |
+| Fixation detection | :heavy_check_mark: | :heavy_minus_sign: | :heavy_check_mark: |
+| Post-hoc pupil detection | :heavy_check_mark:| :heavy_minus_sign: | :heavy_check_mark: |
+| Post-hoc gaze calibration | :heavy_check_mark: | :heavy_minus_sign: | :heavy_check_mark: |
+| Pupil from recording | :heavy_check_mark: | :heavy_minus_sign: | :heavy_minus_sign: |
+| Gaze from recording | :heavy_check_mark: | :heavy_check_mark: | :heavy_minus_sign: |
+
+#### Changelog
```

Comment:
Consider changelog as a info aside instead of a separate section. 

@papr what do you think?

Reply:
I feel like having a title for this section makes sense. Especially since it has a lower level (H4) compared to the section title (H3). This is how it would look like right now:
![Screenshot 2021-07-02 at 10 07 58](https://user-images.githubusercontent.com/168390/124242756-629e2d00-db1d-11eb-879b-5bab6475311c.png)

I would rather remove the citation. This is how it would like without citation and without header:
![Screenshot 2021-07-02 at 10 14 40](https://user-images.githubusercontent.com/168390/124243572-51a1eb80-db1e-11eb-8abe-1f7a22974247.png)